lte/gateway/c/core/oai/tasks/nas/emm/emm_data_ctx.cpp

```diff
@@ -700,22 +700,23 @@ status_code_e emm_context_upsert_imsi(emm_data_t* emm_data,
       (PARENT_STRUCT(elm, struct ue_mm_context_s, emm_context))->mme_ue_s1ap_id;
 
```

Comment:
`h_rc` is not used here
https://github.com/magma/magma/blob/8d67c44a8ab7085551932d5db47b77c89a6f400d/lte/gateway/c/core/oai/tasks/nas/emm/emm_data_ctx.cpp#L698